python/ray/util/annotations.py

```diff
@@ -0,0 +1,76 @@
+def PublicAPI(*args, **kwargs):
+    """Annotation for documenting public APIs.
+
+    Public APIs are classes and methods exposed to end users of Ray. You
+    can expect these APIs to remain backwards compatible even across major Ray
+    releases.
+
+    Args:
+        stability: Either "stable" for stable features or "beta" for APIs that
+            are intended to be public but still in beta.
```

Comment:
For experimental/unstable, is the idea that they just live in `ray.experimental`?

Reply:
Yep, or with comments. We could explore adding a stability="experimental" level as well in the future as needed.